src/components/Page/Page.styles.tsx

```diff
@@ -10,17 +10,22 @@ export const StyledSiteLogo = styled(SiteLogo)`
   left: 3.125rem;
 `;
 
-export const StyledPage = styled.div`
+type StyledPageProps = {
+  isOpen: boolean;
+};
+
+export const StyledPage = styled.div<StyledPageProps>`
   display: flex;
   justify-content: center;
-  align-items: flex-start;
+  align-items: center;
   position: relative;
   min-width: 18rem;
   min-height: 100vh;
   padding: 2rem 0;
+  transition: padding-right 0.3s ease-in-out;
```

Comment:
This not a performant way to animate. Ideally you want to be animating (transitioning) either `transform` or `opacity`. There's some info [here](https://web.dev/animations-guide/).

Same applies to the wallet positioning (which I think has `transition: margin-right`).
On my currently overworked macbook, the wallet and swap widget aren't particularly smooth.

It's definitely fine for the alpha, but something we should address before public beta (which will also have a mobile layout, so should help with some of the layout issues you might face if you switched to `transition: transform` now).